samples/nrf_desktop/configuration/nrf52840_pca20041_ZDebug.conf

```diff
@@ -36,6 +36,7 @@ CONFIG_DESKTOP_BLE_BOND_REMOVAL=y
 CONFIG_DESKTOP_BLE_BOND_REMOVAL_BUTTON=0x0002
 
 CONFIG_DESKTOP_BLE_ADVERTISING_ENABLE=y
+CONFIG_DESKTOP_BLE_DIRECT_ADV=y
```

Comment:
Add only for ZDebug configuration ?

Reply:
Well spotted - I forgot to add others. Thanks!